### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by jeyvison**

### CODE REVIEW

**Comment by jeyvison** [comment]
ci:test:relevant

**Comment by jeyvison** [comment]
ci:test:sf

**Comment by jeyvison** [comment]
ci:test:relevant

**Comment by jeyvison** [comment]
ci:test:relevant

**Comment by jeyvison** [comment]
ci:test:relevant

**Comment by jeyvison** [comment]
ci:test:relevant

**Comment by jeyvison** [comment]
ci:test:relevant

**Comment by jeyvison** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-api/src/main/java/com/liferay/data/engine/model/DEDataLayout.java [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+		if (name == null) {
+			name = new HashMap<>();
+		}
+
+		_name = name;
+	}
+
+	/**
+	 * Sets the pagination mode to this layout. A valid pagination mode must be
+	 * provided in order to save the DataLayout. It must be Wizard or
```
Comment (gabrielwas) [inline modules/apps/data-engine/data-engine-api/src/main/java/com/liferay/data/engine/model/DEDataLayout.java]: @jeyvison incomplete sentence

**Comment by jeyvison** [comment]
ci:test:relevant

**Pull request closed without merging**